pages/search.js

```diff
@@ -1,5 +1,6 @@
 import { useEffect, useState, useCallback, useRef } from "react";
 import { useRouter } from "next/router";
+import { usePathname, useSearchParams } from "next/navigation";
```

Comment:
we are using nextjs pages router ```next/navigation``` is for app router. Please use router from ```next/router```

Reply:
Thanks 👍 remove `next/navigation` and made necessary changes